pkg/sloop/ingress/kubewatcher.go

```diff
@@ -241,15 +332,30 @@ func (i *kubeWatcherImpl) getResourceAsJsonString(kind string, obj interface{})
 	return string(bytes), nil
 }
 
+func (i *kubeWatcherImpl) refershCrdInformers(masterURL string, kubeContext string) {
```

Comment:
should it be 'refreshCRDInformers' ?

Reply:
as mentioned elsewhere, CRD is being used as a 'word' instead of an acronym